cognite/client/_api/assets.py

```diff
@@ -35,7 +36,8 @@ def __call__(
             chunk_size (int, optional): Number of assets to return in each chunk. Defaults to yielding one asset a time.
             name (str): Name of asset. Often referred to as tag.
             parent_ids (List[int]): No description.
-            root_ids (List[Dict[str, Any]]): List of root ids/external ids to filter on.
+            root_ids (Union[List[int], List[Dict[str, Any]], optional): List of root ids ids to filter on. Also accepts a list of mixed id/external id objects of the form [{"id":1},{"externalId":"str"}] for backwards compatibility.
```

Comment:
Don't think we should document the fact that you can pass a list of objects. We will want to deprecate the behaviour at some point, so we don't want people to actually use it, only keep support for backwards compatibility. So remove the type hints aswell.

Reply:
ok